.aptly.conf

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
What is aptly for?

Reply:
It's the repo creator/updater for deb.

Reply:
A third party tool? Do we have to download anything? Is it built in to GoReleaser?

Reply:
It's installed automatically inside of the docker image during the build process. The only tool that we'll need to download is s3-repo-utils.

I'll update our package manager 1-pager w/ a description of the process.

Reply:
Added a bullet list description of the steps and some notes on why. I added the link to the 1 pager in the pr description for convenience.